Kernel/VM/MemoryManager.cpp

```diff
@@ -133,6 +135,25 @@ void MemoryManager::unmap_memory_after_init()
     //Processor::halt();
 }
 
+void MemoryManager::unmap_ksyms_after_init()
+{
+    ScopedSpinLock mm_lock(s_mm_lock);
+    ScopedSpinLock page_lock(kernel_page_directory().get_lock());
+
+    auto start = page_round_down((FlatPtr)&start_of_kernel_ksyms);
+    auto end = page_round_up((FlatPtr)&end_of_kernel_ksyms);
+
+    // Unmap the entire .ksyms section
+    for (auto i = start; i < end; i += PAGE_SIZE) {
+        auto& pte = *ensure_pte(kernel_page_directory(), VirtualAddress(i));
+        pte.clear();
+        flush_tlb(&kernel_page_directory(), VirtualAddress(i));
+    }
+
+    dmesgln("Unmapped {} KiB of kernel symbols after init! :^)", (end - start) / KiB);
+    //Processor::halt();
```

Comment:
o_O

Reply:
Yeah, that's gone. I've also removed it from `unmap_text_after_init`. :)